src/coredump/coredump.c

```diff
@@ -322,18 +329,58 @@ static int make_filename(const Context *context, char **ret) {
         return 0;
 }
 
+static int get_cgroup_memory_limit(uint64_t *ret_max_size) {
+        _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;
+        _cleanup_free_ char *unit = NULL, *unit_path = NULL;
+        uint64_t unit_memory_available;
+        int r;
+
+        assert(ret_max_size);
+
+        r = sd_pid_get_unit(0, &unit);
+        if (r < 0)
+                return log_error_errno(r, "Failed to get current unit name: %m");
+
+        r = sd_bus_path_encode("/org/freedesktop/systemd1/unit", unit, &unit_path);
+        if (r < 0)
+                return log_error_errno(r, "Failed to encode unit %s DBUS path: %m", unit);
+
+        r = sd_bus_default_system(&bus);
+        if (r < 0)
+                return log_error_errno(r, "Failed to connect to system bus: %m");
+
+        r = sd_bus_get_property_trivial(
+                        bus,
+                        "org.freedesktop.systemd1",
+                        unit_path,
```

Comment:
`/org/freedesktop/systemd1/unit/self` should work too

Reply:
it does, updated